docs/src/pages/api-reference/react.mdx

```diff
@@ -56,9 +56,9 @@ export const OurUploadButton = () => (
 
 Config object
 
-| Attribute    | Type    | Required | Notes           | Description                                    |
-| :----------- | :------ | :------- | :-------------- | :--------------------------------------------- |
-| disablePaste | boolean | No       | Added in `v5.7` | disables ability to paste files from clipboard |
+| Attribute     | Type    | Required | Notes           | Description                                   |
+| :------------ | :------ | :------- | :-------------- | :-------------------------------------------- |
+| appendOnPaste | boolean | No       | Added in `v5.7` | enables ability to paste files from clipboard |
```

Comment:
sidenote, if values have a default, we should include that info in this table

Reply:
It's not like it has default. Like if it is not set to true, it will be undefined. Such value is falsy. The paste handler will return if `appendOnPaste` is falsy. Not exactly 'false'.

Should we still clarify, that this feature is not going to be enabled by default?

Reply:
I think it should stay disabled by default, and then I think docs are fine as is now.

We can enable it by default in the next major maybe?

Reply:
The functionality will not work until appendOnPaste is set to true. So it's disabled by default